src/audio/volume/volume_hifi3_with_peakvol.c

```diff
@@ -21,7 +21,9 @@ LOG_MODULE_DECLARE(volume_hifi3, CONFIG_SOF_LOG_LEVEL);
 
 #include "volume.h"
```

Comment:
GIven this seems like a generic problem with volume hifi4 version, I'd actually prefer this over #9221 , but the commit message doesn't match with code. There's no change to "sof/app" and this code is no way specitic to Intel MTL. So if we merge this, the git commit needs to be modified to something like:

```
- app: board: Set CONFIG_VOLUME_HIFI3=y for MTL platform
- Changed VOLUME_HIFI4 to VOLUME_HIFI3 on the MTL platform.
- There was a problem with test
+ audio: volume: disable HIFI4 optimizations
+ There was a problem with test
```